config/environment.js

```diff
@@ -172,6 +172,7 @@ module.exports = function(environment) {
         home: {
             youtubeId: '2TV21gOzfhw',
         },
+        secondaryNavbarId: 'secondaryNavbar',
```

Comment:
Ok just one last tiny thing, I swear! I'd feel safer if this ID were more uniquey, like `__secondaryNavbar` or `secondaryNavbar__squigglepoop` or even a random uuid -- anything that definitely won't be used by accident. Since it's defined in one place and given a name, it can be as ugly as we want without sacrificing readability.